src/guides/v2.4/config-guide/cli/config-cli-subcommands-catalog-sync.md

```diff
@@ -0,0 +1,28 @@
+---
+group: configuration-guide
+title: Catalog Sync
+---
+
+Magento compiles catalog data into tables using indexers. This process is automatically [triggered by several events](https://docs.magento.com/user-guide/system/index-management-events.html), such as changes to a product's price or inventory levels. To allow some Magento services to use that catalog data, a catalog sync process runs hourly. The catalog sync process exports product data from the Magento server to SaaS services on an ongoing basis. For example, the [Product Recommendations](https://docs.magento.com/user-guide/marketing/product-recommendations.html) feature needs up-to-date catalog information so that it can accurately return recommendations with correct names, pricing, and availability. Use the command-line interface to trigger the catalog sync and reindex product data for consumption by SaaS services.
+
+The commands used to sync catalog data are part of the `magento/saas-export` module. See [Install and configure Product Recommendations]({{ page.baseurl }}/recommendations/install-configure.html) for more information.
```

Comment:
`magento/saas-export` looks like a package, which is a [set of modules](https://github.com/magento/saas-export/tree/master/app/code/Magento).